source/type/type.cpp

```diff
@@ -608,6 +608,22 @@ std::vector<idx_t> Network::get(Type_e type, const PT_Data& data) const {
     switch(type) {
     case Type_e::Line: return indexes(line_list);
     case Type_e::Impact: return data.get_impacts_idx(get_impacts());
+    case Type_e::CommercialMode:
+    case Type_e::PhysicalMode:{
+         std::set<idx_t> tmp_result;
```

Comment:
Actually, I don't think we should get the line's physical mode, I think this is the job of the pt ref graph